gantry/firmware/utility_gpio.c

```diff
@@ -1,42 +1,56 @@
 #include "common/firmware/utility_gpio.h"
 #include "platform_specific_hal_conf.h"
 
-
 /**
- * @brief Limit Switch GPIO Initialization Function
+ * @brief LED GPIO Initialization Function
  * @param None
  * @retval None
  */
-void limit_switch_gpio_init(void) {
+void LED_drive_gpio_init(void) {
     /* GPIO Ports Clock Enable */
-    __HAL_RCC_GPIOC_CLK_ENABLE();
+    __HAL_RCC_GPIOB_CLK_ENABLE();
 
-    /*Configure GPIO pin : PC2 */
+    /*Configure GPIO pin : PB11 */
     GPIO_InitTypeDef GPIO_InitStruct = {0};
-    GPIO_InitStruct.Pin = GPIO_PIN_2;
+    GPIO_InitStruct.Pin = GPIO_PIN_11;
+    GPIO_InitStruct.Mode = GPIO_MODE_OUTPUT_PP;
+    GPIO_InitStruct.Pull = GPIO_PULLDOWN;
+    GPIO_InitStruct.Speed = GPIO_SPEED_FREQ_LOW;
+    HAL_GPIO_Init(GPIOB, &GPIO_InitStruct);
+}
+void sync_drive_gpio_init() {
+    /* GPIO Ports Clock Enable */
+    __HAL_RCC_GPIOB_CLK_ENABLE();
+
+    /*Configure GPIO pin : PB7:sync in*/
+    GPIO_InitTypeDef GPIO_InitStruct = {0};
+    GPIO_InitStruct.Pin = GPIO_PIN_5 | GPIO_PIN_7;
```

Comment:
Check with EE before doing this - on X, pin 5 is the tmc2130 clock line and we should make sure it's ok to leave it floating